jujugui/static/gui/src/app/assets/javascripts/react-click-outside.js

```diff
@@ -21,10 +21,23 @@ function enhanceWithClickOutside(WrappedComponent) {
     componentDidMount: function componentDidMount() {
       this.__wrappedComponent = this.refs.wrappedComponent;
       document.addEventListener('click', this.handleClickOutside, true);
+      // XXX ant 9 Feb 2016 Add Esc keybind to also close the current component
```

Comment:
I'm not so sure about this - in an effort to keep this wrapper simple I'd say that the logic as to what to do when a key is down would be up to the `handleKeyDown` method in the component